tests/wpt/web-platform-tests/dom/nodes/Document-createEvent.html

```diff
@@ -151,4 +151,8 @@
     }, 'Should throw NOT_SUPPORTED_ERR for pluralized non-legacy event interface "' + eventInterface + 's"');
   }
 });
+
+test(function() {
+  assert_equals(document.createEvent('storageevent'), "");
+}, "document.createEvent('storageevent') does not propagate the document's URL.")
```

Comment:
Yes. If you want to add a regression test for this specific fix, please do so in wpt/mozilla instead.